R/demo-food-access.R

```diff
@@ -1,19 +1,22 @@
 #' Access to Food Insecurity Data
 #' 
-#' This function returns a data frame from Food Insecurity in State Cancer Profiles
+#' This function returns a data frame about Food Insecurity from State Cancer Profiles.
```

Comment:
Lowercase for "Food Insecurity" and similar, it's not a proper noun. (State Cancer Profiles is a proper noun so leave it capitalized).